dev/small-requirements.txt

```diff
@@ -8,3 +8,4 @@ pytest==3.2.1
 pytest-cov==2.6.0
 pytest-localserver==0.5.0
 moto==1.3.16
+azure-storage-blob>=12.0.0
```

Comment:
@WeichenXu123 To clarify, we've added this as a test requirement because it's required for `azure_blob_artifact_repo.py` right?

Reply:
Yes!